workloads/devstack-xenial.yaml

```diff
@@ -0,0 +1,55 @@
+---
+inherits: xenial
+needs_nested_vm: true
+vm:
+  mem_mib: 6144
+  cpus: 4
+  ports:
+  - host: 55580
+    guest: 80
+...
+---
+{{- define "ENV" -}}
+{{proxyVars .}}
+{{- print " DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true " -}}
+{{end}}
+{{ define "DEVSTACKPATH" }}{{ if $.MountPath "devstack" }}{{$.MountPath "devstack"}}{{else}}/home/{{.User}}/devstack{{end}}{{end}}
+#cloud-config
+write_files:
+ - content: |
+     [url "http://"]
+       insteadOf=git://
+   path: /home/{{.User}}/.gitconfig
+   permissions: '0755'
+
+runcmd:
+  - chown -R {{.User}}:{{.User}} /home/{{.User}}/
```

Comment:
This probably won't be needed if you don't use write_files.

Reply:
+1. Dumped this and moved all file creation to happen as user {{.User}}